### PR TITLE
Update forms direct deposit check guide image with SVG

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Description
This PR replaces the default direct deposit check guide image with a higher quality SVG and removes the previous image from the project.

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/34712

## Testing done
N/A

## Screenshots
Screen Shot 2022-01-06 at 13 48 26

## Acceptance criteria
- [x] Replace forms default direct deposit check guide image with SVG
- [x] Remove direct deposit check guide PNG image from project

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms-system/src/js/definitions/content/directDeposit.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const directDepositDescription = () => {
   return (
     <div className="vads-u-margin-top--2">
       <img
-        src="/img/direct-deposit-check-guide.png"
+        src="/img/direct-deposit-check-guide.svg"
```
Comment (timwright12) [inline src/platform/forms-system/src/js/definitions/content/directDeposit.jsx:11]: is there a separate PR open in content-build to add this image?

Comment (andresriveratoro) [reply src/platform/forms-system/src/js/definitions/content/directDeposit.jsx:11]: Hi Tim, [here](https://github.com/department-of-veterans-affairs/content-build/pull/886) is the PR for content-build.

Comment (k80bowman) [reply src/platform/forms-system/src/js/definitions/content/directDeposit.jsx:11]: We would need to merge the content build PR first, correct? Will it fail the vets-website build if we don't?

Comment (timwright12) [reply src/platform/forms-system/src/js/definitions/content/directDeposit.jsx:11]: the build won't fail, but the image will be broken. I commented on the content-build PR

**Review by timwright12 — APPROVED** [review]

**Review by pdavies88 — APPROVED** [review]
Able to confirm locally on form:
Screen Shot 2022-01-07 at 2 50 51 PM

File: src/platform/forms-system/src/js/definitions/content/directDeposit.jsx (hunk 1 above)
Comment (timwright12) [reply src/platform/forms-system/src/js/definitions/content/directDeposit.jsx:11]: Confirmed that this image is in prod now http://content.www.va.gov.s3-website-us-gov-west-1.amazonaws.com/img/direct-deposit-check-guide.svg

**Review by timwright12 — APPROVED** [review]

**Review by k80bowman — APPROVED** [review]

**Pull request merged**